### PR TITLE
added display/hide functionality to the sidebar for mobile view

### PR DESCRIPTION
**Pull request by vidhi-thakur**
image image

please check preview URL [here] (https://deploy-preview-3--notes-savvy.netlify.app/dashboard)

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code structure looks good and the live preview (https://deploy-preview-3--notes-savvy.netlify.app/dashboard) also works as expected and is looking really awesome @vidhi-thakur .  
Faced a minor issue which I faced while toggling between mobile modes and desktop screen mode 
Whenever I used to hide the sidebar in mobile view then go back to the desktop view the sidebar wasn't visible and so was the hamburger icon so what I can suggest is that you can have a hamburger icon on both the mobile and desktop views that can help you keep up the sidebar in show or hidden state whatever desired!!
Another thing which you can add is a _redirects file in the public folder of your repo having the following contents:
```
/*    /index.html 200
```
So by  adding this file one can avoid the PAGE Not found errors coming up from Netlify end either on refreshing the screens with routes those errors will not come up

**Comment by vidhi-thakur** [comment]
> The code structure looks good and the live preview (https://deploy-preview-3--notes-savvy.netlify.app/dashboard) also works as expected and is looking really awesome @vidhi-thakur . Faced a minor issue which I faced while toggling between mobile modes and desktop screen mode Whenever I used to hide the sidebar in mobile view then go back to the desktop view the sidebar wasn't visible and so was the hamburger icon so what I can suggest is that you can have a hamburger icon on both the mobile and desktop views that can help you keep up the sidebar in show or hidden state whatever desired!! Another thing which you can add is a _redirects file in the public folder of your repo having the following contents:
> 
> ```
> /*    /index.html 200
> ```
> 
> So by adding this file one can avoid the PAGE Not found errors coming up from Netlify end either on refreshing the screens with routes those errors will not come up

Thank you for you review, have made the changes

File: src/components/sidebar/Sidebar.css [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
-    padding: var(--xxl-spacing) var(--default-spacing) 0;
-    width: min(18.75rem, 70vw);
-    min-height: 100vh;
-    background-color: var(--basic-bg);
-    z-index: -1;
+  padding: var(--xxl-spacing) var(--default-spacing) 0;
+  width: min(18.75rem, 70vw);
+  min-height: 100vh;
+  background-color: var(--basic-bg);
+  z-index: -1;
```
Comment (anik31) [inline src/components/sidebar/Sidebar.css:6]: appreciation - liked the way you have declared the css properties

**Review by ApurvaSawant11 — APPROVED** [review]
Everything looks good to me!

Just a UX suggestion, you can implement it later: 
When the user resizes the window, keep the sidebar inactive by default. The focus should be on the notes and not the sidebar. 

Another point of view: Suppose the user logs in from mobile.  The first thing that's visible should be the notes and not the sidebar.

**Review by anik31 — APPROVED** [review]
Everything looks good. The UI & code structure is very nice. The hamburger menu works fine & looks good on mobile view.

**Comment by vidhi-thakur** [comment]
Thank ypu @anik31 @ApurvaSawant11  for the reviews

**Pull request merged**